File: ass-ade-v1.1/tests/fixtures/rebuilt-out/bridges/swift/Sources/AssAdeBridge/a0_qk_constants/BridgeContract.swift
import Foundation

let defaultManifestRel = ".ass-ade/bridges/bridge_manifest.json"

struct BridgeManifestSummary {
    let schema: String
    let bridgeReady: Bool
    let bridgeLanguages: [String]
    let pythonBridgeCommand: [String]
}

File: ass-ade-v1.1/tests/fixtures/rebuilt-out/bridges/rust/src/a0_qk_constants/bridge_contract.rs
#[derive(Debug, Clone)]
pub struct BridgeManifestSummary {
    pub schema: String,
    pub bridge_ready: bool,
    pub bridge_languages: Vec<String>,
    pub python_bridge_command: Vec<String>,
}

pub const DEFAULT_MANIFEST_REL: &str = ".ass-ade/bridges/bridge_manifest.json";
